### ISSUE TITLE
Conflict

### ISSUE THREAD

**Issue by Sephirox** [NekoNova/ItemPreviewImproved#3]
Addon does not work when a NeedVGreed addon is in use as 
http://www.curse.com/ws-addons/wildstar/222352-tloot

**Comment by coding-red-panda** [comment]
Correct, if Addons modify the existing Need vs Greed Window, I cannot guarantee ItemPreviewImprovedEx to be working, as I try to hook into this window.
As long as they keep the same structured window setup the Addon will work, regardless if the error message is there.

It works for example with ForgeUI, despite telling so.
I will see if these can work together, but putting this on low prio.
